### PR TITLE
Cleaning rejected petition view by removing redundant info and also add new styling

### PR DESCRIPTION
**Pull request by mybits**
- removed: the signature count, confirmation email section, references to 10k and 100k sections,
created by section

changed: 
* Change the date from deadline to be the date created, and call it Date submitted.
* add the styled 'flash' message
* add a reason for rejection header

PT: https://www.pivotaltracker.com/n/projects/307301/stories/96936096

### CODE REVIEW

File: app/views/petitions/_rejected_petition.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<h1>
+  <span class="heading-secondary">Rejected petition</span>
+  <%= @petition.title %>
```
Comment (alexander-bobin) [inline app/views/petitions/_rejected_petition.html.erb]: Is it better to have `petition` is a local variable explicitly passed through or just rely on it being available as `@petition`? I'm not sure

File: app/views/petitions/show.html.erb [hunk 2]
```diff
@@ -12,69 +12,62 @@
...
-<% end if @petition.closed? %>
+<% if @petition.rejected? %>
+  <%= render 'rejected_petition' %>
+<% else %>
 
-<h1>
-  <span class="heading-secondary">Petition</span>
-  <%= @petition.title %>
-</h1>
+  <%= content_tag :p, :class => 'flash-notice' do -%>
```
Comment (alexander-bobin) [inline app/views/petitions/show.html.erb]: Is it better to have this as another partial too? Maybe someone else has some reckons

File: app/views/petitions/_rejected_petition.html.erb (hunk 1 above)
Comment (h-lame) [inline app/views/petitions/_rejected_petition.html.erb]: Providing a partial local param gets my vote.  Relying on globals is a bad idea.

**Comment by otlaitil** [comment]
:+1:

File: app/views/petitions/_rejected_petition.html.erb (hunk 1 above)
Comment (mybits) [inline app/views/petitions/_rejected_petition.html.erb]: Thanks @alexander-bobin and @h-lame! Couldn't agree more, fixed here:  https://github.com/alphagov/e-petitions/commit/943535ae3adbc57ce531958fcac954c733c70f60

File: app/views/petitions/show.html.erb (hunk 2 above)
Comment (mybits) [inline app/views/petitions/show.html.erb]: You mean the heading? If so, I'm not sure if that wouldn't make the code less readable actually..

Comment (alexander-bobin) [inline app/views/petitions/show.html.erb]: Nope. I meant that we have a partial which represents a rejected petition. Maybe we should have a partial that represents open or not-rejected. It's more of a question to see what others think.

Comment (h-lame) [inline app/views/petitions/show.html.erb]: It might make it clearer if all the stuff in the ``else`` branch was in a partial, yeah.

**Comment by mybits** [comment]
just been :recycle:

File: app/views/petitions/show.html.erb (hunk 2 above)
Comment (mybits) [inline app/views/petitions/show.html.erb]: Ok, will do! Any suggestion how to name this partial? I could only come with names like `view_petition`,  `petition_show`, `not_rejected_petition` - and I don't think they're good enough :thought_balloon:

Comment (alexander-bobin) [inline app/views/petitions/show.html.erb]: Ah naming. I'd either call it `_petition.html.erb` or `_show_petition.html.erb` and use `_show_rejected_petition.html.erb`

Comment (mybits) [inline app/views/petitions/show.html.erb]: Like the `petition_show` and `rejected_petition_show` idea :) Will implement those.

**Comment by mybits** [comment]
@alexander-bobin, @h-lame and @otlaitil thanks for your comments! I've added the additional partial in the last commit.

**Comment by alexander-bobin** [comment]
LGTM :shipit:

**Comment by davidbasalla** [comment]
looks good! :+1:

**Pull request merged**